clusterHost.py

```diff
@@ -89,10 +89,11 @@ def _create_k8s_nodes(configs: list[NodeConfig]) -> list[ClusterNode]:
     def _k8s_nodes(self) -> list[ClusterNode]:
         return self.k8s_master_nodes + self.k8s_worker_nodes
 
-    def _ensure_images(self, local_iso_path: str, infra_env: str, nodes: list[ClusterNode]) -> None:
+    def ensure_images(self, local_iso_path: str, infra_env: str, masters: bool) -> None:
         if not self.hosts_vms:
             return
```

Comment:
Is this ever needed if ensure_images becomes a helper function?

Reply:
I don't it's guaranteed that the node has any VMs and that assumption didn't change.